### PR TITLE
Revert "Revert "Revert "Revert "Revert "remove two unused couch views…

### PR DESCRIPTION
**Pull request by dannyroberts**
Reverts dimagi/commcare-hq#8368

### CODE REVIEW

**Comment by dannyroberts** [comment]
:cry:

File: corehq/apps/hqadmin/_design/views/domains_over_time/map.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+function(doc) {
```
Comment (dimagimon) [inline corehq/apps/hqadmin/_design/views/domains_over_time/map.js]: Missing name in function declaration.

File: corehq/couchapps/reports_forms/views/all_submitted_users/map.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+function(doc) {
```
Comment (dimagimon) [inline corehq/couchapps/reports_forms/views/all_submitted_users/map.js]: Missing name in function declaration.

File: corehq/couchapps/reports_forms/views/all_submitted_users/map.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+    // !code util/reports_forms.js
+    if(doc.doc_type == "XFormInstance" && get_user_id(doc) != null) {
```
Comment (dimagimon) [inline corehq/couchapps/reports_forms/views/all_submitted_users/map.js]: Use '!==' to compare with 'null'.

**Pull request merged**